hier.c

```diff
@@ -243,3 +227,22 @@ void hier()
   c(auto_qmail,"man/man8","qmail-command.8",auto_uido,auto_gidq,0644);
   c(auto_qmail,"man/cat8","qmail-command.0",auto_uido,auto_gidq,0644);
 }
+
+void hier_queue()
```

Comment:
I would put this above ```hier()```, and then just call it from where it was extracted. That way you don't have to modify the files that call ```hier()```, which will behave just as before.